Add missing filters info and corrected info on some modules

TheRealPear opened the pull request:
This pull request adds some missing information concerning the `pulse`, `countdown`, `after`, and `players` filter. Additionally, I corrected some information regarding Ender chests and relabeled the values for some items (formatted message instead of strings, etc.) and some editorial changes.

Pablete1234 commented on docs/modules/blocks/enderchests.mdx:
```diff
@@ -85,7 +84,7 @@ Additionally, players cannot place Ender chests under any circumstances.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Region</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```
Needs to be "Randomize-able" Region, not all bounded regions will work.
As an example, a sphere region (which is bounded) won't work.

Pablete1234 commented on docs/modules/blocks/enderchests.mdx:
```diff
@@ -157,7 +156,7 @@ Additionally, players cannot place Ender chests under any circumstances.
         </td>
         <td>
-          <a href="/docs/modules/mechanics/regions">Bounded Regions</a>
+          <a href="/docs/modules/mechanics/regions">Bounded Region</a>
```
given that the region & filter are defined as properties, there's no reason to re-repeat the same thing in a second table. the whole "Dropoff Sub-elements" table can go

TheRealPear replied:
This is mostly precedent from other pages having a similar layout. [Block Drops](https://pgm.dev/docs/modules/blocks/blockdrops), [Falling Blocks](https://pgm.dev/docs/modules/blocks/falling-blocks), [Portals](https://pgm.dev/docs/modules/mechanics/portals)

Pablete1234 replied:
i thought the standard was to list them just in the attributes region with a "property" tag, if it seems more common to list them in both places then that's fine, but i think some other places will need expansion as not all attributes marked as property have another table for the children

TheRealPear replied:
I'll remove it from the enderchest page for now and clean up the rest of the pages in a future pull request

TheRealPear replied:
![image](https://github.com/PGMDev/Website/assets/20259871/5a169154-a669-4d69-8c50-30da470c3dcb)

Pablete1234 commented on docs/modules/mechanics/actions-triggers.mdx:
```diff
@@ -493,7 +533,15 @@ trigger an Action.
+          >
+            Property
+          </span>
+          Sets a previously defined action.
```
*previously defined* made sense when you could only reference by id, maybe this text can be tweaked
`Sets the action to run when the filter allows`

Pablete1234 commented on docs/modules/mechanics/filters.mdx:
````diff
@@ -1216,11 +1292,15 @@ This filter is a [Dynamic Filter](#dynamic-filters).
 ```
 <filters>
-    <!-- Pulses activation for 15 seconds in a 60 second time period -->
+    <!-- Pulses activation for 15 seconds in a 60 seconds time period -->
````
Pretty sure the former was correct? "60 second time period" sounds more correct than with the s

TheRealPear commented:
Just a last minute change here, I have no idea how soon this will be reviewed. I've updated Fandom links, once again, to the new Minecraft wiki (https://minecraft.wiki/), which will be the new home for Minecraft information moving forward, with less obstructive ads and better experience for visitors.

I've rebased and unified the existing commits into one for easier reviewing. Let me know if there is anything that needs correcting

Pablete1234 commented on docs/modules/blocks/enderchests.mdx:
```diff
@@ -9,12 +9,11 @@ The `<enderchest>` tag can contain any number of `<dropoff>` tags. Each `<dropof
 
 Dropoffs handle taking a player's Ender chest inventory and dropping it on the ground at a defined region or it can completely erase it.
-This can be triggered with a filter or once the player switches team or leaves the server.
+This is triggered once the player switches team or leaves the server.
```
Technically if its they leave the match, or "stop participating" in the match (eg: /obs), but its quite a small detail i think most people wont notice the difference

TheRealPear replied:
Reworded to `This is triggered once the player switches teams or stops participating in the match.`

CoWinkKeyDinkInc commented:
Thanks for changing the wiki links.
